pixi.toml

```diff
@@ -0,0 +1,28 @@
+[project]
+name = "iris-hep.github.io"
+version = "0.1.0"
+description = "Jekyll based website for IRIS-HEP"
+authors = ["Matthew Feickert <matthew.feickert@cern.ch>"]
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-64", "osx-arm64", "win-64"]
+
+[tasks]
+# Need to use '--user-install' as using pixi instead of rbenv
+install = """
+gem install --user-install bundle && \
```

Comment:
The use of `--user-install` results in things getting installed under `./.pixi/envs/default/`.